IdentityCore/src/MSIDConstants.h

```diff
@@ -94,6 +94,14 @@ typedef void (^MSIDGetDeviceInfoRequestCompletionBlock)(MSIDDeviceInfo * _Nullab
 @compatibility_alias MSIDViewController NSViewController;
 #endif
 
+#if TARGET_OS_IPHONE
+#import <UIKit/UIKit.h>
```

Comment:
Question: Do we also need passing window for iOS? Since we are using UIViewController for iOS. Thanks!

Reply:
Can we combine this code into lines 89-94?

Reply:
Updated